.github/workflows/ci.yml

```diff
@@ -219,6 +219,30 @@ jobs:
       - name: Run bindgen tests
         run: cargo test -p spacetimedb-cli
 
+  publish_checks:
+    name: Check that packages are publishable
+    runs-on: ubuntu-latest
+    permissions: read-all
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python env
+        run: |
+          test -d venv || python3 -m venv venv
+          venv/bin/pip3 install argparse toml
+      - name: Run checks
+        run: |
+          FAILED=0
+          # This definition of ROOTS and invocation of find-publish-list.py is copied from publish-crates.sh
+          ROOTS=(bindings sdk cli standalone)
+          for crate in $(venv/bin/python3 tools/find-publish-list.py --recursive --quiet "${ROOTS[@]}"); do
+            if ! venv/bin/python3 tools/crate-publish-checks.py "crates/$crate"; then
+              FAILED=$(( $FAILED + 1 ))
+            fi
+          done
+          if [ $FAILED -gt 0 ]; then
+            exit 1
```

Comment:
Can we include a message in here about what this means if it fails? What do we need to do if this workflow fails?

Reply:
It looks like this when it fails:
![image](https://github.com/user-attachments/assets/04ccb0e1-9ab9-442d-9712-4d56c1a2ec2c)
Do you think that's enough, or should we add an extra `echo` at the end to sum up?

Reply:
Maybe I could improve the message rather than `Non-path dev-dependencies found...` to be a bit clearer?

Reply:
> Maybe I could improve the message rather than `Non-path dev-dependencies found...` to be a bit clearer?

yeah I think if we improved the clarity on that it would look great 👍

Reply:
What do you think of this?
![image](https://github.com/user-attachments/assets/388a021c-7c60-4666-a998-496fa22ac045)

Reply:
Much better 👍